types/stripe/index.d.ts

```diff
@@ -6734,6 +6750,108 @@ declare namespace Stripe {
         }
     }
 
+    namespace taxRates {
+        interface ITaxRateCreationOptions {
+          /**
+           * The display name of the tax rate, which will be shown to users.
+           */
+            display_name: string;
```

Comment:
nit: the indentation for these values comments doesn't align with the comments. I think you have an extra space everywhere.